sequencer/src/bin/espresso-dev-node.rs

```diff
@@ -735,18 +749,48 @@ pub struct ApiState {
     pub wallet: EthereumWallet,
     /// L1 chain id
     pub l1_chain_id: u64,
+    /// The HotShot event streamer for the first node
+    pub event_streamer: Option<Arc<RwLock<EventsStreamer<SeqTypes>>>>,
```

Comment:
I think this whole thing can be removed if we just enable hotshot events module here 
```
 let api_options = options::Options::from(options::Http {
        port: sequencer_api_port,
        max_connections: sequencer_api_max_connections,
    })
    .submit(Default::default())
    .config(Default::default())
    .explorer(Default::default())
    .query_sql(Default::default(), sql)
    .hotshot_events(Default::default())
  ;
```